snippets/freemarker.snippets

```diff
@@ -70,4 +70,5 @@ snippet att "<#attempt>...<#recover></#attempt>"
 	</#attempt>
 # Then built-in for booleans
 snippet ?th "?then(true, false)"
-	?then(${1:true}, ${0:false${VISUAL}})
\ No newline at end of file
+	?then(${1:true}, ${0:false${VISUAL}})
+
```

Comment:
It looks like there's 2 newlines here now, could you remove the last one so there's no empty line please? Thanks

Reply:
Ok, so I don't understand what I'm suppose to do. I've opened the file with Vim, gone to the end of the last line (filled with snippet body) and hit return. That of course creates a new empty line at the end of the file...

Reply:
Previously the file ended with `	?then(${1:true}, ${0:false${VISUAL}})`, now it ends with `	?then(${1:true}, ${0:false${VISUAL}})\n\n`. I'll fix it :)

Reply:
Ok, thank you.